pkg/kubectl/cmd/convert/import_known_versions.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package convert
+
+// These imports are the API groups the client will support.
+// TODO: Remove these manual install once we don't need legacy scheme in kubectl
+import (
```

Comment:
Why is this needed?

Reply:
this is the tricky part: as the last dependency for internal client is gone, the legacyscheme which is used by convert would be empty unless we manually install internal types. and thats the purpose of this file. if u notice, we are doing the same in `cmd/kube-controller-manager` for the exact same reason. w/o this file, the convert wont be working b/c legacyscheme doesn't know how to deal w/ conversions.